samples/data-grid/cypress/grouped-headers/demo.js

```diff
@@ -0,0 +1,55 @@
+window.dataGrid = DataGrid.dataGrid('container', {
+    table: {
+        columns: {
+            id: ['1', '2', '3', '4'],
+            product: ['Apples', 'Pears', 'Plums', 'Bananas'],
+            weight: [100, 40, 0.5, 200, 100, 40, 0.5, 200, 100],
+            price: [1.5, 2.53, 5, 4.5],
+            url: ['http://path1.to', 'http://path2.to', 'http://path2.to', 'http://path3.to'],
+            icon: ['Apples URL', 'Pears URL', 'Plums URL', 'Bananas URL']
+        }
+    },
+    settings: {
+        header: [
+            {
+                columnId: 'id'
+            },
+            {
+                headerFormat: 'Product',
+                columns: [{
+                    headerFormat: 'Product name',
```

Comment:
It isn't applied to the grid
I've tested with:
```ts
                columns: [{
                    headerFormat: 'Product name test',
                    columnId: 'product'
                }, {
```

Reply:
Because you have already 
<img width="333" alt="Screenshot 2024-08-16 at 13 37 28" src="https://github.com/user-attachments/assets/92a67a19-55db-4618-ba14-438957c02a38">
defined headerFormat in columsn section:

Reply:
Yes, but header should take precedence if defined there.